File: Python/Iniciante/Solucao_1036.py
import math

def calcular_delta(a,b,c):#função para calcular o delta
    delta  = pow(b,2) - 4 * a * c
    return delta

def calcular_bhaskara(a,b,c,delta):#função para calcular bhaskara
    r1 = (-b+math.sqrt(delta)) / (2 * a)
    r2 = (-b-math.sqrt(delta)) / (2 * a)
    raizes = [r1,r2]#retorna uma lista com as duas raizes
    return raizes

a, b, c = map(float,input().split())# aqui funcionar da seguinte forma map pega e transforma tudo em float split pega todos os
#elementos a partir de um limitador no caso aqui o espaço e jogar dentro de um array
#a, b e c estão recebendo os valores via desetruturação
delta = calcular_delta(a,b,c)
j = 1

if a == 0 or delta < 0:
    print("Impossivel calcular")
else:
    raizes = calcular_bhaskara(a,b,c,delta)

    for i in raizes:
        print(f"R{j} = {i:.5f}")
        j += 1

File: Java/src/Iniciante/Solucao_1036.java
import java.io.IOException;
import java.util.Scanner;

public class Solucao_1036 {

    public static double delta(double a, double b, double c) { // metodo para calcular o delta
        double delta = Math.pow(b, 2) - 4 * a * c;
        return delta;
    }

    public static double[] calcularBhaskara(double a, double b, double c, double delta) {//metodo para calcular as raizes com retorno em um array para retorna tanto
        double r1 = (-b + Math.sqrt(delta)) / (2 * a);//quando a raiz delta for positivo
        double r2 = (-b - Math.sqrt(delta)) / (2 * a);//quanto negativo
        double[] raizes = {r1, r2};
        return raizes;
    }


    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        double[] raizes = new double[2];
        double a = input.nextDouble();
        double b = input.nextDouble();
        double c = input.nextDouble();
        boolean flag = true;
        double delta = delta(a, b, c);


        if ( a == 0 || delta < 0 ) {//ser a  igual a 0 ou delta e nagativo então não existem raizes
            System.out.println("Impossivel calcular");
        } else {
            raizes = calcularBhaskara(a, b, c, delta);//chamo a metodo
            int i = 1;//contador para imprimir R1 e R2
            for (double raiz : raizes){
                System.out.printf("R%d = %.5f%n", i, raiz);
                i++;
            }
        }
        input.close();//fechamento do input
    }
}